Makefile

```diff
@@ -1,17 +1,18 @@
 # TARGETS
-charm: clean ## Build the charm
-	charmcraft pack
-	cp jobbergate-cli_ubuntu-20.04-amd64_centos-7-amd64.charm jobbergate-cli.charm
+charm: clean version ## Build the charm
+	@charmcraft pack
+	@cp jobbergate-cli_ubuntu-20.04-amd64_centos-7-amd64.charm jobbergate-cli.charm
 
 lint: ## Run linter
 	tox -e lint
 
+version: ## Create/update version file
+	@git describe --tags --dirty --always > version
```

Comment:
I'm not sure about the `--tag` here since both nextgen and legacy share the same repo, and we will be using tags for the nextgen. I wonder if this may lead to misunderstandings, what do you think?

Reply:
I think we should define a suffix for the legacy tags. For example: `1.0.0-legacy`. What do you think about that?

Reply:
Good point, I see no problem with that